internships/gsoc2022.md

```diff
@@ -1,80 +1,115 @@
-
-The interview can have the following three outcomes:
-* Conditional fast-track acceptance to the sktime GSoC 2022. This means you are one of the three top ranking candidates. This means highly likely - but not guaranteed - acceptance to the GSoC program, but can still result in a rejection (see below).
-* Conditional acceptance to the sktime GSoC 2022. This means you are on the ranked list put forward to sktime.  This means likely - but not guaranteed - acceptance to the GSoC program, but can still result in a rejection (see below).
-* A rejection for GSoC 2022. This means you are not put forward to any of the GSoC 2022 slots by the sktime team - but we would still very much welcome collaboration and contribution, outside of GSoC 2022, if you are interested.
-
-The interview outcome will be communicated to you no later than one week after the interview itself. You will receive full confirmation of acceptance or rejection (from sktime and GSoC) on May 20, directly by GSoC.
-
-## What we are looking for
-We're actively looking for contributors and your help is extremely welcome. Therefore, if
- * you are interested in time series, machine learning (ML), statistics, API design and software architecture,
- * you like coding in Python,
- * you are familiar with the basic data science ecosystem in Python, including numpy, pandas and scikit-learn, 
- * you enjoy working with a vibrant team of experienced ML scientists and software engineers,
- * you always wanted to join an open-source community,
+## Why sktime?
+Time series data is ubiquitous in many applications. Examples include sensor readings from industrial processes, spectroscopy wave length data from chemical samples, or bed-side monitor medical data from patients. Developing advanced time series analysis capabilities for researchers and practitioners is one of the major challenges of contemporary machine learning.
+
+sktime is a new Python toolbox for machine learning with time series and, to the best of our knowledge, the first unified toolbox for time series. Our ambition is to provide for time series what scikit-learn provides for tabular data. This involves extending scikit-learn to the different time series learning tasks, such as time series classification, clustering, forecasting and anomaly detection. To find out more, check out our [paper](http://learningsys.org/neurips19/assets/papers/sktime_ml_systems_neurips2019.pdf) published at the [Workshop on Systems for ML at NeurIPS 2019](http://learningsys.org/neurips19/).
+
+## The application process
+
+There are two main phases: the sktime application and GSoC proposal, and an interview. Details of these are below. The timeline is:
```

Comment:
mentions "two main phases", but then lists three items. Confusing.
Timeline afterwards does not map clearly onto the phases.